Sources/VaporSQLite/Provider.swift

```diff
@@ -1,51 +1,54 @@
 import Vapor
 import FluentSQLite
 
-public typealias MySQLDriver = FluentSQLite.SQLiteDriver
-
 public final class Provider: Vapor.Provider {
-    public var provided: Providable
 
     
     public enum Error:Swift.Error {
         case noSQLiteConfig
         case pathMissing
     }
-    /**
-        MySQL database driver created by the provider.
-    */
-    public let driver: SQLiteDriver
-
-    /**
-        MySQL database created by the provider.
-    */
-    public let database: Database?
-
-    public init(
-        path: String
-    ) throws {
-        let driver = try SQLiteDriver(path: path)
-        let database = Database(driver)
-
-        self.driver = driver
-        self.database = database
-        
-        provided = Providable(database: database)
-    }
     
-    public convenience init(config: Config) throws {
+    private(set) public var driver: SQLiteDriver!
```

Comment:
Hi, my first opinion was that since the driver was not accessible from the Droplet directly, it should be made available here, but now that I think about it, yes it is : through `droplet.database.driver`.
So it should be removed.